android_default_browser_metrics.view.lkml

```diff
@@ -0,0 +1,77 @@
+
+view: android_default_browser_metrics {
+  derived_table: {
+    sql: WITH dau_segments AS 
+          (SELECT submission_date, SUM(DAU) as dau
+          FROM `moz-fx-data-shared-prod.telemetry.active_users_aggregates` 
+          WHERE app_name = 'Fenix'
+          --AND channel = 'release'
+          AND submission_date >= '2021-01-01'
+          GROUP BY 1),
+          
+      
+      product_features AS
+      (SELECT
+          client_info.client_id,
+          DATE(submission_timestamp) as submission_date,
+          COALESCE(SUM(CASE WHEN metrics.boolean.metrics_default_browser THEN 1 ELSE 0 END)) AS metrics_default_browser
+          FROM `mozdata.fenix.metrics`
+      where DATE(submission_timestamp) >= '2021-01-01'
+      AND sample_id = 0
+      group by 1,2),
+      
+      product_features_agg AS
+      (SELECT
+          submission_date,
+          --metrics_default_browser
+          100*COUNT(DISTINCT CASE WHEN metrics_default_browser > 0 THEN client_id END) AS metrics_default_browser_users,
+          100*COALESCE(SUM(metrics_default_browser), 0) AS metrics_default_browser,
+      FROM product_features
+      where submission_date >= '2021-01-01'
```

Comment:
this is probably why things are taking so long. looking all the way back to 2021.